### PR TITLE
First project commit

### PR DESCRIPTION
**Pull request by Lemi-Agrey**
This is a pull request for the signup page

### CODE REVIEW

File: routes/userRoute.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        res.status(500).json({
+            error : err
+        })
+    })
+})
+
+module.exports=router
+
+
+
```
Comment (jennizalwango) [inline routes/userRoute.js]: Remove these last empty lines

File: index.js [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+        }
+    });
+});
+
+
+const PORT=process.env.PORT || 4000;
+
+app.listen(PORT, () => {
+    console.log(`Server listening on port http://localhost:${PORT}`);
+});
```
Comment (jennizalwango) [inline index.js:54]: Add a new line to this file

File: models/userModel.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+    last_name: {type: String, required: true},
+    email: {type: String, 
+        required: true, 
+        unique: true,
+        match: /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/
+    },
+    password: {type: String, required: true}
+});
+
+module.exports=mongoose.model('User', userSchema);
```
Comment (jennizalwango) [inline models/userModel.js:16]: Add a new line this file @Lemi-Agrey

**Review by jennizalwango — APPROVED** [review]
Good Work @Lemi-Agrey But look at the comments then we will merge

File: models/userModel.js (hunk 3 above)
Comment (Lemi-Agrey) [reply models/userModel.js:16]: I have not understood. you mean i new empty line

**Pull request merged**